samples/src/main/java/org/opensearch/client/samples/RandUtil.java

```diff
@@ -0,0 +1,23 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+package org.opensearch.client.samples;
+
+public class RandUtil {
```

Comment:
Probably `knn` package is better place, I suspect we would have more samples added